README.rst

```diff
@@ -92,10 +92,9 @@ the ``WATCHMAN_CHECKS`` setting. In ``settings.py``::
         'another.module.path.to.callable',
     )
 
-Checks now have the same contract as context processors: they consume a
-``request`` and return a ``dict`` whose keys are applied to the JSON response::
```

Comment:
Cool, those were pretty similar to my thoughts on things. Thanks for your input!